src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,64 @@
+$cell-side: 100px;
+$cell-sideAndGap: $cell-side + 1;
+$weekDays: (
+  "mon": 0,
+  "tue": $cell-sideAndGap * 1,
+  "wed": $cell-sideAndGap * 2,
+  "thu": $cell-sideAndGap * 3,
+  "fri": $cell-sideAndGap * 4,
+  "sat": $cell-sideAndGap * 5,
+  "sun": $cell-sideAndGap * 6
+);
+
+.calendar {
+  &__list {
+    display: flex;
+    gap: 1px;
+    padding: 10px;
+    margin: 0;
+    box-sizing: border-box;
+    list-style: none;
+    flex-wrap: wrap;
+    width: $cell-side * 7 + 20px + 6px;
+
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: 100px;
+    height: 100px;
+    border: solid 1px #000;
+    box-sizing: border-box;
+    transition: transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Why are you altering `cursor` here?

Reply:
When you are pointing to the day it becomes pink and moves 20px up + cursor becomes pointer in order to make user experience even better (without this feature browser treat each day as a text and offers to copy it)

Reply:
Oh no actually it works well even with a common cursor, however I like more when it is pointer, do I have to change it?

Reply:
Anyway, I have already changed it